mkdocs.yml

```diff
@@ -86,11 +86,13 @@ nav:
           - Compute resources: configuration/resources.md
           - Suspending scans: configuration/suspend-scan.md
           - Retain issues: configuration/retain-issues.md
+          - HTTPS proxy: configuration/https-proxy.md
           - Ignore unfixed vulnerabilities: plugins/trivy/#large-vulnerability-reports
-          - HTTPS Proxy: configuration/https-proxy.md
-          - Scanning Images hosted in AWS ECR: configuration/aws-elastic-container-registry.md
-          - Scanning Images hosted in Azure ACR: configuration/acr.md
           - Vulnerability scan timeout: plugins/trivy/#scan-timeout
+          - Private registries:
```

Comment:
We should differentiate between private registries (for AWS/Azure) and Authenticated Registries.

Reply:
Done